### PR TITLE
Add three more unit tests for compute_eval_from_predictions

### PR DESCRIPTION
**Pull request by Lguyogiro**
There were some lines of my previous PR (making `compute_eval_from_predictions` support probabilities in predictions file) that were not covered by tests. This PR adds three tests for those cases.

### CODE REVIEW

File: tests/test_utilities.py [hunk 1]
```diff
@@ -202,11 +240,12 @@ def test_compute_eval_from_predictions_with_probs():
...
 @raises(ValueError)
 def test_compute_eval_from_predictions_breaks_with_expval_and_nonnumeric_classes():
     """
-    Make sure compute_eval_from_predictions breaks when predictions are calculated via
-    expected_value and the classes are non numeric.
+    Make sure compute_eval_from_predictions breaks with ValueError when predictions are
+    calculated via expected_value and the classes are non numeric.
     """
 
     pred_path = join(_my_dir, 'other',
```
[the remark this answers is not in the document]
Comment (Lguyogiro) [reply tests/test_utilities.py]: Not sure what you mean. It is the same path (with a different filename) used in all of the `test_compute_eval_from_predictions` tests.

[the remark this answers is not in the document]
Comment (desilinguist) [reply tests/test_utilities.py]: I am seeing just the following. Is that just GitHub being weird?

```
 pred_path = join(_my_dir, 'other',
```

[the remark this answers is not in the document]
Comment (Lguyogiro) [reply tests/test_utilities.py]: Yes I believe it is. The file name is on the following line. The two lines together read:
```
    pred_path = join(_my_dir, 'other',
                     'test_compute_eval_from_predictions_nonnumeric_classes_predictions.tsv')
```

[the remark this answers is not in the document]
Comment (jbiggsets) [reply tests/test_utilities.py]: I'm also seeing the full path:

```
 pred_path = join(_my_dir, 'other', 'test_compute_eval_from_predictions_nonnumeric_classes_predictions.tsv')
```

**Pull request merged**